client/index.html

```diff
@@ -122,75 +122,81 @@ <h1 class="title">Sign in to The Lounge</h1>
 						<div class="row">
 							<div class="col-sm-12">
 								<h1 class="title">
-									<%= public ? "The Lounge - " : "" %>
+									{{ternary public "The Lounge " " " }}
```

Comment:
Looks like a typo here. I think you meant

```handlebars
{{ternary public "The Lounge - " "" }}
```
or
```handlebars
{{#if public}}The Lounge - {{/if}}
```
which I prefer as more native (and also find more readable due to the quotes, but maybe that's personal).